src/ucs/debug/debug.c

```diff
@@ -654,11 +656,16 @@ static void ucs_debugger_attach()
             argv[narg++] = "-p";
             argv[narg++] = pid_str;
         }
+        /* Resolve where our temporary dir is located. */
+        tmp_dir = getenv("TMPDIR");
```

Comment:
how about adding a function
```
static const char * ucs_get_tmpdir()
```
which would check env var, and return /tmp by default